_includes/configuration/image-file-types.md

```diff
@@ -0,0 +1,20 @@
+### `image_file_types`
+
+This option allows you to specify which image file formats will be recognised and modified by [smart_paste](#smart_paste) when pasted.
+
+**Type:** `String`
+
+**Default Value:** `'jpeg,jpg,jpe,jfi,jfif,png,gif,bmp,webp'`
+
+**Possible Values:** A list of valid web image file formats. For a list of possible values see: [Image file type and format guide](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types).
```

Comment:
Should this mention "extensions"? as that's what this setting accepts.

Reply:
some people call them extensions, so please add this.

Reply:
Also, prefix the link text with `MDN Web Docs - `
important to make clearer that they are leaving our site for this info.